experimental/uefi/loader/src/qemu.rs

```diff
@@ -71,6 +71,10 @@ impl Qemu {
         // Construct the command-line arguments for `qemu`. See
         // https://www.qemu.org/docs/master/system/invocation.html for all available options.
 
+        // Needed to expose advanced CPU features to the UEFI app. Specifically
+        // RDRAND which is required for remote attestation.
+        cmd.arg("-enable-kvm");
+        cmd.args(&["-cpu", "Broadwell-IBRS"]);
```

Comment:
Any particular reason for `Broadwell-IBRS`? Granted, I don't know what the default CPU that qemu emulates is, but if you want something with RDRAND I think you can explicitly ask for that with `+rdrand`, something like "`-cpu host,+rdrand`". I think explicitly listing that we require rdrand is better than hiding it in the CPU definition.

That being said, qemu docs are not the best, so please do check the syntax for the `-cpu` flag to see how it works.

Reply:
> Any particular reason for Broadwell-IBRS?

No strong reason for the particular model, other than it supports RDRAND while being old enough that it can hopefully be virtualized by most hosts. This particular line just matches what we went with in https://github.com/project-oak/oak/pull/2703. 

In general we do probably want to specify a specific model. The reason is that in virtualization mode, qemu does not appear to use a default CPU. Instead it's either host passthrough (unstable between hosts, not recommended by docs) or a named model. :) 

See https://qemu-project.gitlab.io/qemu/system/qemu-cpu-models.html#two-ways-to-configure-cpu-models-with-qemu-kvm for more details